docs/core/fallback-actions.rst

```diff
@@ -8,87 +8,74 @@ Fallback Actions
 
 .. edit-link::
 
+.. contents::
+   :local:
+
 Sometimes you want to revert to a fallback action, such as replying,
-`"Sorry, I didn't understand that"`. You can handle fallback cases by adding
-either the ``FallbackPolicy`` or the ``TwoStageFallbackPolicy`` to your
-policy ensemble.
+`"Sorry, I didn't understand that"`. You can handle fallback cases by adding appropriate
+rules. Rasa Open Source comes with two default implementations for handling these
+fallbacks.
+In addition you can also use :ref:`custom-actions` to implement custom procedures.
 
-Fallback Policy
----------------
+Handling Low NLU Confidence
+---------------------------
 
+Although Rasa's :ref:`intent-classifier` will generalize to unseen messages, some
+messages might still receive a low classification confidence.
+In order to handle messages which have low confidence, we recommend to add the
+:ref:`fallback-classifier` to your NLU pipeline. The :ref:`fallback-classifier` will
+predict an intent ``nlu_fallback`` whenever no other intent prediction crosses
+the configured confidence threshold.
 
-The ``FallbackPolicy`` has one fallback action, which will
-be executed if the intent recognition has a confidence below ``nlu_threshold``
-or if none of the dialogue policies predict an action with
-confidence higher than ``core_threshold`` or if the highest ranked intent differs in
-confidence with the second highest ranked intent by less than ``ambiguity_threshold``.
+Writing Stories / Rules for Messages with Low Confidence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The thresholds and fallback action can be adjusted in the policy configuration
-file as parameters of the ``FallbackPolicy``.
+When you added the :ref:`fallback-classifier` to your NLU pipeline, you can treat
+messages with low classification confidence just as any other intent. The following
+:ref:`rule<rules>` will ask the user to rephrase whenever they send a message which is
+classified with low confidence:
 
 .. code-block:: yaml
 
-  policies:
-    - name: "FallbackPolicy"
-      nlu_threshold: 0.4
-      core_threshold: 0.3
-      ambiguity_threshold: 0.1
-      fallback_action_name: "action_default_fallback"
+    - rule: Ask the user to rephrase whenever they send a message with low NLU confidence
+      steps:
+      - ...
```

Comment:
this will be updated later?

Reply:
You mean with the `condition: ` step? Yes, all the changes from the feedback has to be worked in.

Reply:
yes, so I meant, should it be part of this PR, or part of condition PR

Reply:
The condition PR will definitely go in before this one, so I'd say this PR.